cmake/std/atdm/ats2/environment.sh

```diff
@@ -242,7 +242,11 @@ export MPICC=`which mpicc`
 export MPICXX=`which mpicxx`
 export MPIF90=`which mpif90`
 
-export ATDM_CONFIG_MPI_POST_FLAGS="-disable_gpu_hooks;-map-by;socket:PE=4"
+export ATDM_CONFIG_MPI_EXEC=${ATDM_SCRIPT_DIR}/ats2/trilinos_jsrun
+
+# TODO: Need -np support from the trilinos_jsrun wrapper.
+export ATDM_CONFIG_MPI_POST_FLAGS="--rs_per_socket 4"
+export ATDM_CONFIG_MPI_EXEC_NUMPROCS_FLAG="-p"
```

Comment:
@jjellio: Note these configuration options for MPI (or jsrun in this case). I've tested the changes introduced in this PR using your `trilinos_jsrun` wrapper. Based on the results of those tests, it looks like we're still missing the logic for handling GPU-based tests in the wrapper.